plugins/micromega/sos.ml

```diff
@@ -338,7 +338,7 @@ let token s =
   many (some isspace) ++ word s ++ many (some isspace) >> fun ((_, t), _) -> t
 
 let decimal =
-  let ( || ) = parser_or in
+  let ( ||| ) = parser_or in
```

Comment:
Bisect got confused with this overloading, I guess we should see if that's the case and submit a bug there (and remove this change)